src/databricks/labs/dqx/llm/pk_identifier.py

```diff
@@ -0,0 +1,665 @@
+"""
```

Comment:
remove all these comments, this is suppose to be a module, not a standalone tool